org.jacoco.core/src/org/jacoco/core/internal/flow/Instruction.java

```diff
@@ -102,11 +121,11 @@ public int getBranches() {
 	}
 
 	/**
-	 * Returns the number of covered branches starting from this instruction.
-	 * 
-	 * @return number of covered branches
+	 * Returns coverage of branches of this instruction.
+	 *
+	 * @return coverage of branches of this instruction
 	 */
-	public int getCoveredBranches() {
+	public BitSet getCoveredBranches() {
```

Comment:
Of course internal to the internal class 😉 . For whitebox testing of `Instruction` we can still expose the `BitSet` but for interaction with other classes classes I would prefer a more semantic interface (like `merge()`).